### PR TITLE
Fix error message in `mc admin subnet health`

### PR DESCRIPTION
**Pull request by anjalshireesh**
Change from

`Please use 'mc support diagnostics'`

to

`Please use 'mc support diag'`

### CODE REVIEW

**Review by harshavardhana — CHANGES_REQUESTED** [review]
while we are at this we should also support dummy flags until we fully deprecate these commands to ensure that we can throw appropriate message such as `mc support diag --flag`

**Comment by anjalshireesh** [comment]
> while we are at this we should also support dummy flags until we fully deprecate these commands to ensure that we can throw appropriate message such as `mc support diag --flag`

Have updated the PR to support all flags supported by `mc support diag`. The behavior now will be:

```
> ./mc admin subnet health --offline
mc: Please use 'mc support diag'
```

**Comment by anjalshireesh** [comment]
> > while we are at this we should also support dummy flags until we fully deprecate these commands to ensure that we can throw appropriate message such as `mc support diag --flag`
> 
> Have updated the PR to support all flags supported by `mc support diag`. The behavior now will be:
> 
> ```
> > ./mc admin subnet health --offline
> mc: Please use 'mc support diag'
> ```

Updated to include arguments and flags also in the new command

File: cmd/admin-subnet-health.go [hunk 1]
```diff
@@ -29,10 +33,33 @@ var adminSubnetHealthCmd = cli.Command{
 	Action:             mainSubnetHealth,
 	Before:             setGlobalsFromContext,
 	Hidden:             true,
-	CustomHelpTemplate: "Please use 'mc support diagnostics'",
+	Flags:              append(supportDiagFlags, globalFlags...),
+	CustomHelpTemplate: "Please use 'mc support diag'\n",
```
Comment (ravindk89) [inline cmd/admin-subnet-health.go:37]: Should we be more specific?

"This command is deprecated and will be removed in a future release. Use `mc support diag` instead."

Comment (anjalshireesh) [reply cmd/admin-subnet-health.go]: Done

**Review by harshavardhana — APPROVED** [review]

**Review by ravindk89 — APPROVED** [review]

**Pull request merged**